src/main/java/seedu/address/logic/commands/BackCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_RECORDS;
+
+import seedu.address.model.Model;
+
+/**
+ * Displays all persons or records after a find or rfind command is executed
+ * Keyword matching is case insensitive.
+ */
+public class BackCommand extends Command {
```

Comment:
Maybe will be good to enforce it such that back is used for reverting the list from a find command in both the patient list or record list, while the list command is used to transition back from the record list to the patient list. This would prevent the overlap in the usage of the list command.

Reply:
Agreed, would help to standardise the workflow by enforcing that the user must call `back` after using `find` / `rfind` to return to the original unfiltered list as opposed to allow them to call `list` / `rlist` to "relist" the displayed list.

Reply:
Good point, will update to reflect changes.